Sources/TSAuthenticationSDK.xcframework/ios-arm64/TSAuthenticationSDK.framework/Headers/TSAuthenticationSDK.swift

```diff
@@ -8,162 +8,148 @@
      */
-    public func executeWebauthnRegistration(completion: TSAuthenticationResponseCompletion = nil) {
-        TSAuthenticationImpl.shared.executeWebauthnRegistration(completion: completion)
+    public func authenticate(username: String, completion: TSAuthenticationCompletion = nil) {
+        TSAuthenticationImpl.shared.authenticate(username: username, completion: completion)
     }
-
-
-    /**
-     Obtains the authentication challenge from the Transmit Service, and stores the response locally to be used by `executeWebauthnAuthentication`. This allows reducing latency when authentication is requested. To provide the best experience, this should be called as soon as possible.
-
-     This call must be invoked for a registered user. It is invoked on the last user that logged in or registered unless you specify a different user in the request (for example, when explicitly switching to a different user). If the target user is not registered or in case of any other failure, an error will be returned.
-    */
-    public func prepareWebauthnAuthentication(username: String, completion: TSAuthenticationCompletion = nil) {
-        TSAuthenticationImpl.shared.prepareWebauthnAuthentication(username: username, completion: completion)
+    
+    
+    private func registerNativeBiometrics(username: String, completion: @escaping TSNativeBiometricsRegistrationCompletion) {
+        Task {
+            await TSAuthenticationImpl.shared.registerNativeBiometrics(username: username, completion: completion)
+        }
     }
-
-
+    
+    private func authenticateNativeBiometrics(username: String, challenge: String, completion: @escaping TSNativeBiometricsAuthenticationCompletion) {
+        Task {
+            await TSAuthenticationImpl.shared.authenticateNativeBiometrics(username: username, challenge: challenge, completion: completion)
+        }
+    }
+    
     /**
-     Invokes a WebAuthn authentication, including prompting the user for biometrics. This must be called only after calling `prepareWebauthnAuthentication` to obtain the server challenge.
-
-     If authentication is completed successfully, this call will return a promise that resolves to an authorization code, which can be used to obtain user tokens. If it fails, an error will be returned instead.
+     Invokes a WebAuthn credential sign transaction, including prompting the user for biometrics.
+     If transaction signing is completed successfully, this function will return a callback containing a WebAuthnEncodedResult.
+     The WebAuthnEncodedResult should be used to make a completion request using your backend API which will commuincate with Transmit's Service
      */
-    public func executeWebauthnAuthentication(completion: TSAuthenticationResponseCompletion = nil) {
-        TSAuthenticationImpl.shared.executeWebauthnAuthentication(completion: completion)
+    public func signTransaction(username: String, completion: TSAuthenticationCompletion = nil) {
```

Comment:
@igorbabitski I thing we should change it to signWebauthnTransaction

Reply:
@asaftransmitsecurity got it, is it means that we need to release a new version?